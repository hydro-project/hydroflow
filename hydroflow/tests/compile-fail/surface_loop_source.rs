fn main() {
    let mut df = hydroflow::hydroflow_syntax! {
        loop {
            source_iter(0..10) -> null();
        }
    };
    df.run_available();
}
