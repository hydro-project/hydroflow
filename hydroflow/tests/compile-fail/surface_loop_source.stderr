error: Source operator `source_iter(...)` must be at the root level, not within any `loop { ... }` contexts.
 --> tests/compile-fail/surface_loop_source.rs:4:13
  |
4 |             source_iter(0..10) -> null();
  |             ^^^^^^^^^^^^^^^^^^
